examples/kubernetes/1.10/cilium-containerd-ds.yaml

```diff
@@ -106,6 +106,7 @@ spec:
           capabilities:
             add:
             - NET_ADMIN
+            - SYS_MODULE
           privileged: true
```

Comment:
Maybe we should remove this line? As far as I know, `privileged: true` adds **all** capabilities that exist in the kernel. I think it's better to have only listed capabilities.

Reply:
@tgraf I recall there being discussion that over some time we'd want to do exactly this (remove the `privileged: true` line, but that we are not there yet? I.e., we'll remove this in a future release after some more testing?

Reply:
Alright, I'm fine with that. :+1: I was just curious and I thought that for some reason you need to define both `privileged: true` and list specific caps.